centreon/src/Core/HostCategory/Infrastructure/Repository/DbReadHostCategoryRepository.php

```diff
@@ -104,10 +104,25 @@ public function findAll(?RequestParametersInterface $requestParameters): array
         $concatenator = new SqlConcatenator();
         $concatenator->withCalcFoundRows(true);
         $concatenator->defineSelect(
-            'SELECT hc.hc_id, hc.hc_name, hc.hc_alias, hc.hc_activate, hc.hc_comment
+            'SELECT DISTINCT hc.hc_id, hc.hc_name, hc.hc_alias, hc.hc_activate, hc.hc_comment
             FROM `:db`.hostcategories hc'
         );
 
+        if (! empty($_GET['search']) && mb_strpos($_GET['search'], 'host_group_id')) {
```

Comment:
Do not use $_GET directly, the REquestParametersInterface has a method to do this check

https://github.com/centreon/centreon/blob/develop/centreon/src/Centreon/Domain/RequestParameters/Interfaces/RequestParametersInterface.php#L52